utils/python/CIME/SystemTests/homme.py

```diff
@@ -62,3 +61,12 @@ def run_phase(self):
             run_cmd_no_fail("%s -j 4 check >& %s" % (gmake, log), from_dir=exeroot)
         else:
             run_cmd_no_fail("%s -j 4 baseline >& %s" % (gmake, log), from_dir=exeroot)
+
```

Comment:
@mvertens did you have something in mind? This code is specific to the ACME HOMME test hack. I don't CESM cares about this change.